### DISCUSSION TITLE
Using the library on a NUCLEO-L432KC card (256K)

### DISCUSSION THREAD

**Discussion by ArkaIison** [ArminJo/Talkie#32] [Q&A] [answered]
Hello,
I am a final year engineering student working on an IoT weather station. I would like to know if it is possible to use the Talkie library for the NUCLEO-L432KC card.
Thanks in advance for the answer!

**Comment by ArminJo** [comment]
You can try to use the BluePill settings for your board, it could be working.

**Comment by radiohound** [comment, marked as answer]
I have tried it on a Nucleo-H746ZG and it worked. Only thing is, the sound was coming out A0. So you might test a piezo on that pin.

**Reply by ArminJo** [reply]
Thanks for the feedback.
